src/python/grpcio/grpc/_cython/_cygrpc/aio/common.pyx.pxi

```diff
@@ -67,3 +67,23 @@ class _EOF:
 
 
 EOF = _EOF()
+
+
+class BaseError(Exception):
+    """The base class for all exceptions generated by gRPC framework."""
```

Comment:
Unfortunately, this statement is aspiration, not fact. We need to make it less strong, since we still raise lots of other errors in the sync stack.

Reply:
Rephrased to `The base class for exceptions generated by gRPC AsyncIO stack.`